numba_cuda/numba/cuda/tests/cudapy/test_extending.py

```diff
@@ -35,15 +35,17 @@ def sum_intervals(i, j):
     from numba.core import cgutils
     from numba.core.extending import (
         lower_builtin,
-        make_attribute_wrapper,
         models,
-        register_model,
         type_callable,
         typeof_impl,
     )
     from numba.core.typing.templates import AttributeTemplate
     from numba.cuda.cudadecl import registry as cuda_registry
     from numba.cuda.cudaimpl import lower_attr as cuda_lower_attr
+    from numba.cuda.extending import (
+        register_model,
```

Comment:
@gmarkall I've update this test by importing `register_model` from `numba.cuda.extending`, so later in this file `make_attribute_wrapper` works on the model registered only in `numba.cuda`. However I agree, that we need to add a test that `make_attribute_wrapper` is called on the struct that uses types both from cuda dmm and default dmm

Reply:
Okay, it has `types.float64`, that uses default dmm. So no need for extra tests.